nens/Plot breach data.ipynb

```diff
@@ -0,0 +1,177 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot breach data\n",
+    "\n",
+    "Using this script you can plot data regarding breaches in a 3Di model. Plots include breach discharge, velocity, width, depth, waterlevel on both sides of the breach.\n",
+    "\n",
+    "- Author: Emiel Verstegen, 2019\n",
+    "\n",
+    "- email: emiel.verstegen@nelen-schuurmans.nl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#import libraries, make sure you have them installed on your Python environment\n",
+    "from threedigrid.admin.gridresultadmin import GridH5ResultAdmin\n",
+    "from threedigrid.admin.gridadmin import GridH5Admin\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import math\n",
+    "from pyproj import Proj, transform\n",
+    "from google_drive_downloader import GoogleDriveDownloader as gdd #Needs installing using pip, unlike the others"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#download testdata and unzip in testdata directory (https://drive.google.com/open?id=1xNhMZnWKxFP7on1reXxecq7XdTuJCSa2)\n",
+    "\n",
+    "\n",
+    "gdd.download_file_from_google_drive(file_id='1xNhMZnWKxFP7on1reXxecq7XdTuJCSa2',\n",
+    "                                    dest_path='./testdata/bergermeer.zip',\n",
+    "                                    unzip=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Define location of raw results and gridadministration\n",
+    "result_path = 'testdata/'\n",
+    "nc = os.path.join(result_path,'results_3di.nc')\n",
+    "f = os.path.join(result_path,'gridadmin.h5')\n",
+    "\n",
+    "#Load files into gr object\n",
+    "ga = GridH5Admin(f)\n",
+    "gr = GridH5ResultAdmin(f,nc)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#subset only 1D nodes\n",
+    "breaches = gr.breaches\n",
+    "breach_lines = gr.lines.filter(kcu__eq=55)"
```

Comment:
Kan je er een regeltje commentaar bij zetten?
- Wat is "kcu"?
- `__eq` betekent "equal" (hm, is dat nodig? is `kcu=55` niet al afdoende)?

Reply:
kcu staat niet specifiek ergens voor (nagevraagt), het is een classifier wat voor soort flowline het betreft. __eq is verwijderd, was inderdaad niet nodig. Commentaar toegevoegd.